### PR TITLE
[SDPA-4688] preview update

### PR DESCRIPTION
**Pull request by tim-yao**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-4688

### Changed

1.  Added update script for release preview module.

### CODE REVIEW

File: packages/create-ripple-app/tidemodules.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
   { value: 'event', name: 'Event', default: true },
   { value: 'news', name: 'News', default: true },
   { value: 'grant', name: 'Grant', default: false },
-  { value: 'gtm', name: 'Google Tag Manager', default: true },
   { value: 'profile', name: 'Profile', default: true },
   { value: 'media', name: 'Media', default: true },
   { value: 'webform', name: 'Webform', default: true },
   { value: 'search', name: 'Search', default: true },
-  { value: 'publication', name: 'Publication', default: false },
+  { value: 'publication', name: 'Publication', default: true },
```
Comment (rubyllamadora) [inline packages/create-ripple-app/tidemodules.js:14]: Is this change required for the preview?

**Review by rubyllamadora — APPROVED** [review]

File: packages/create-ripple-app/tidemodules.js (hunk 1 above)
Comment (tim-yao) [reply packages/create-ripple-app/tidemodules.js:14]: No, it's irrelevant to preview. Just correct the default value for it when I edit this file. It's the create ripple app default settings. Publication should be enabled for all now.
